identity/app/views/consentJourney.scala.html

```diff
@@ -29,16 +31,28 @@
                     <div class="js-errorHolder manage-account__errors"></div>
 
                     <div class="manage-account-wizard manage-account-wizard--repermission" >
-                        <div class="manage-account-wizard__step">
-                            <h1>Welcome to repermissioning</h1>
-                        </div>
                         <div class="manage-account-wizard__step">
 
                             <form novalidate action="@idUrlBuilder.buildUrl("/privacy/edit", idRequest)" role="main" method="post">
                                 @views.html.helper.CSRF.formField
 
-                                <h2 class="manage-account-header">We need you to consent again to receiving communications from The Guardian</h2>
-                                <p class="manage-account-headerNote">These are all the permissions you already consented to, please review them and check them again if you still want to receive them</p>
+                                @if(journey == "repermission") {
+                                    <h1 class="identity-title">
+                                        We need you to consent again to receiving communications from The Guardian
+                                    </h1>
+
+                                    <div class="manage-account-flexrow">
+                                        <p class="manage-account-headerNote manage-account-flexrow_item">Tick the boxes you are interested in. We have featured some you might be interested in.</p>
+                                        <a href="#" class="manage-account__button manage-account-flexrow_item manage-account-flexrow_item--static">
```

Comment:
That should be a button ...

Reply:
really shouldn't, it's a link just doesn't really have a place to point to at the moment because we are missing the page. should be removed altogether